### DISCUSSION TITLE
[Feature Request] Image enlarging on click

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6420] [Ideas]
Upon reading OBS' documentation, I realized this would be really cool if there was a click-to-enlarge feature for images

https://github.com/squidfunk/mkdocs-material/assets/82747632/25b85bd8-226b-4fcc-ab65-0fb197209fd8

The best way to implement this imo would be to take the whole screen (this does not completely to leave some space for the x arrow), and left click anywhere (even on the image) to dismiss

**Comment by copdips** [comment]
try this: mkdocs-glightbox
